packages/contracts/contracts/testing/ERC1155Reward.sol

```diff
@@ -0,0 +1,93 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+import "@openzeppelin/contracts/token/ERC1155/extensions/ERC1155Supply.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+import "@snickerdoodlelabs/erc7529/contract/ERC7529.sol";
+
+contract ERC1155Reward is ERC1155, AccessControl, ERC1155Supply, ERC7529 {
+    bytes32 public constant MINTER_ROLE = keccak256("MINTER_ROLE");
+
+    // Number of reward options
+    uint256 public tokenCount; 
+
+    mapping(uint256 => string) public tokenIdToURI; 
+
+    /// @notice Takes the initial number of rewards offered and sets the current token id
+    constructor(uint256 numberOfRewards, string[] memory tokenURIs) ERC1155("") {
+        _grantRole(DEFAULT_ADMIN_ROLE, msg.sender);
+        _grantRole(MINTER_ROLE, msg.sender);
+        
+        require(numberOfRewards > 0, "ERC1155Reward: Number of rewards must be greater than zero");
```

Comment:
Agreed, in that case we can just remove this require statement because any uint value will work.

Updated to include a check if rewards count is 0, it would not increment token id and do a setURI.